aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1378,7 +1384,7 @@ func createSourceConfig(source map[string]interface{}) *firehose.KinesisStreamSo
 	return configuration
 }
 
-func createS3Config(d *schema.ResourceData) *firehose.S3DestinationConfiguration {
+func createS3Config(d *schema.ResourceData, withErrorOutputPrefix bool) *firehose.S3DestinationConfiguration {
```

Comment:
`s3`/`elasticsearch`/`splunk` support `ErrorOutputPrefix`, however, `redshift` does not